plugin.rb

```diff
@@ -35,6 +35,12 @@
   require 'topic_assigner'
   require 'pending_assigns_reminder'
 
+  class ::WebHookEventType
+    ASSIGN = 101.freeze
```

Comment:
This is a bit difficult to swallow because it means plugins could use the same id.

I know in the past @SamSaffron suggested plugins make PRs to core to reserve ids. It's not great but might be better than this? Sam do you still think that's the best approach?